src/components/Button.js

```diff
@@ -216,7 +216,7 @@ class Button extends Component {
                         {textComponent}
                     </View>
                     {this.props.shouldShowRightIcon && (
-                        <View>
+                        <View style={styles.justifyContentCenter}>
```

Comment:
@osama256 
Sorry, why this was change was made?
For whatever reason Isn't there any alternate solution,  as it could affect all button usage?

cc: @pecanoro

Reply:
@Santhosh-Sellavel We made this change to fix one of [these bugs](https://github.com/Expensify/App/pull/10060#issuecomment-1193350895), `alignItemsCenter` style already used in the sibling `View` 
https://github.com/Expensify/App/blob/22a0cba7aaeaa786f04a63a5ad74c650c4c35001/src/components/Button.js#L207-L220
And since the parent `View` uses `flexRow` style, both children should use `alignItemsCenter` style, so I guess the developer forgot to add it.